web.config

```diff
@@ -19,6 +19,10 @@
 	  <clear />
 	  <rule name="StatusRedirect" stopProcessing="true">
       <match url="(.*)" ignoreCase="true" />
+        <conditions logicalGrouping="MatchAll">
+          <add input="{REQUEST_URI}" negate="true" pattern="^/features/" ignoreCase="true" />
```

Comment:
@kypflug I believe the trailing slash is not needed here and the next condition (we don't enforce them in this project)